modin/core/storage_formats/pandas/parsers.py

```diff
@@ -669,13 +668,13 @@ class PandasParquetParser(PandasParser):
         _doc_parse_func,
         parameters="""files_for_parser : list
     List of files to be read.
-storage_options : dict
-    Parameters for specific storage engine.""",
+""",
     )
-    def parse(files_for_parser, storage_options, **kwargs):
+    def parse(files_for_parser, **kwargs):
         from pyarrow.parquet import ParquetFile
 
         columns = kwargs.get("columns", None)
+        storage_options = kwargs.pop("storage_options", {}) or {}
```

Comment:
```suggestion
        storage_options = kwargs.pop("storage_options", None) or {}
```
this would be sliiiiiiiiiiiiiiiiightly faster (you won't have to construct a dict at each call to `.pop()`, only when it returns you something empty)
or
```suggestion
        storage_options = kwargs.pop("storage_options", {})
```
looks shorter. Pick any you like :)

Reply:
I think I actually need to implement it this way. I encountered the case where `storage_options` wasn't in `kwargs` which then returns the empty dict. If we have `storage_options` that maps to nothing, then we get none, which we don't handle, so we need the second `or {}`.